evap/contributor/views.py

```diff
@@ -28,7 +28,7 @@ def index(request):
     semesters = Semester.objects.all()
     semester_list = [dict(semester_name=semester.name, id=semester.id, courses=[course for course in all_courses if course.semester_id == semester.id]) for semester in semesters]
 
-    template_data = dict(semester_list=semester_list, delegated_courses=delegated_courses)
+    template_data = dict(semester_list=semester_list, delegated_courses=delegated_courses, user=user)
```

Comment:
could you check whether this is necessary? we should automatically have a user variable in templates, if not, we have request.user.